src/cpp/src/continuous_batching/model_runner.hpp

```diff
@@ -424,6 +517,23 @@ class ModelRunner {
 
         _reset_cache_rotation_coefficients();
 
+        if (_is_hs_export()) {
+            try {
```

Comment:
please re-write this code with required conditional checking without exceptions. Exceptions mechanism quite slowdowns performance.